tests/ModelValidationActionFilter_CustomOptionsTests.cs

```diff
@@ -25,7 +25,7 @@ public async Task OnActionExecutionAsync_ControllerDoesNotHaveApiControllerAttri
 
             // Act
             var response =
-                await Client.PostAsJsonAsync($"{ControllerWithoutApiAttributeEndpoint}/test-validator", payload);
+                await HttpClientJsonExtensions.PostAsJsonAsync(Client, $"{ControllerWithoutApiAttributeEndpoint}/test-validator", payload);
```

Comment:
**question**: It's realy a minor issue, but I didn't understand why changing the call to `PostAsJsonAsync` this way. `Client.PostAsJsonAsync` stopped working? If our custom extension is causing problems I think it's better to remove it since we don't need it anymore.